travis/update-docker-hub.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -eo pipefail
+
+docker build -t mitodl/bootcamp_web_travis -f Dockerfile .
```

Comment:
this is a mistake: I copied the script to use it in the future, but I did not update it: I should just remove it